solr_schema_init/create_isb_core_schema.py

```diff
@@ -1,9 +1,13 @@
 import json
 import typing
-
 import requests
+import time
+
+# Sleep for 20 seconds to allow for the Solr Docker container to start up.
+time.sleep(20)
 
-SOLR_API = "http://localhost:8983/api/collections/isb_core_records/"
+CREATE_CORE_API = "http://solr:8983/admin/cores?action=CREATE&name=isb_core_records"
```

Comment:
I never got this part working.  I'll comment it out and add some notes.  I might try messing with it for a bit more, too, as it would be nicer if we didn't have to go through the admin UI to do this.